File: server/weapons.go
package main

type Weapon int

const (
	WeaponNone Weapon = iota
	WeaponKnife
	WeaponPistol
	WeaponTommy
)

type WeaponStats struct {
	Range        float64
	Damage       int
	ReloadTime   int
	AmmoCapacity int
}

var WeaponStatsMap = map[Weapon]WeaponStats{
	WeaponNone:   {},
	WeaponKnife:  {10, 34, 0, 1},
	WeaponPistol: {PlayerMovementRange * 5, 5, 2, 10},
	WeaponTommy:  {PlayerMovementRange * 10, 8, 4, 25},
}

File: player/weapons.py
import enum

from dataclasses import dataclass

@dataclass
class WeaponStats:
    Range: float
    Damage: int
    ReloadTime: int
    AmmoCapacity: int


@dataclass
class WeaponNone:
    stats = WeaponStats(0, 0, 0, 0)
@dataclass
class WeaponKnife:
    stats = WeaponStats(10, 34, 0, 1)
@dataclass
class WeaponPistol:
    stats = WeaponStats(5 * 5, 5, 2, 10)
@dataclass
class WeaponTommy:
    stats = WeaponStats(5 * 10, 8, 4, 25)


weaponTypes = {
    0: WeaponNone,
    1: WeaponKnife,
    2: WeaponPistol,
    3: WeaponTommy
}

itemTypes = {
    0: "weapon",
    1: "health"
}
